src/components/Circles/CreateEditPostModal.tsx

```diff
@@ -94,28 +94,28 @@ export const CreateEditPostModal = () => {
                 title={t('create-post-modal-title', 'Create Post')}
               />
             </Appbar.Header>
-            <TextInput
-              autoFocus
-              testID={tID('post-text-input')}
-              multiline
-              numberOfLines={12}
-              defaultValue={postToEdit?.message}
-              placeholder={t(
-                'create-post-placeholder-text',
-                'What do you want to share?',
-              )}
-              style={styles.textArea}
-              onChangeText={(text: string) => {
-                setPostText(text);
-                setCharacterCount(text.length);
-              }}
-            />
+
             <KeyboardAvoidingView
-              behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
-              keyboardVerticalOffset={Platform.OS === 'ios' ? 20 : 40}
-              style={styles.toolbarContainer}
+              behavior="padding"
```

Comment:
Did you try this on Android too? I recall there was some major hurdle with getting this to work correctly across both platforms but perhaps I missed something here.

Reply:
I did try this on android and it seemed like it was working to me. I think moving the view to wrap all the content with the flex grow on the text view made this behave better on both platforms

Reply:
Interesting, sounds good! Glad it is working 👍